packages/components/modal/examples/ModalBasicExample.tsx

```diff
@@ -1,63 +1,19 @@
 import React, { useState } from 'react';
-import {
-  Button,
-  Modal,
-  FormControl,
-  TextInput,
-} from '@contentful/f36-components';
+import { Button, Modal, Paragraph } from '@contentful/f36-components';
 
 export default function ModalBasicExample() {
   const [isShown, setShown] = useState(false);
-  const [contentTypeName, setContentTypeName] = useState('');
+
   return (
     <>
-      <Button onClick={() => setShown(true)}>Create new content type</Button>
-      <Modal
-        onClose={() => setShown(false)}
-        shouldCloseOnOverlayClick
-        shouldCloseOnEscapePress
-        isShown={isShown}
-      >
+      <Button onClick={() => setShown(true)}>Open modal</Button>
+      <Modal onClose={() => setShown(false)} isShown={isShown}>
         {() => (
           <>
-            <Modal.Header
-              title="Create new content type"
-              onClose={() => setShown(false)}
-            />
+            <Modal.Header title="Modal title" onClose={() => setShown(false)} />
             <Modal.Content>
-              <FormControl>
-                <FormControl.Label isRequired>Name</FormControl.Label>
-                <TextInput
-                  maxLength={20}
-                  value={contentTypeName}
-                  placeholder="For example Product, Blog Post, Author"
-                  onChange={(e) => {
-                    setContentTypeName(e.target.value);
-                  }}
-                />
-              </FormControl>
+              <Paragraph>Modal content is here!</Paragraph>
```

Comment:
```suggestion
              <Heading>First entry published! It can now be fetched via the APIs</Heading>
               <Paragraph>
                  To discover more about how to consume content from the APIs, go to Space Home.
               </Paragraph>
```

I found a modal in the webapp with this content, maybe it would be cool to have something more "real case"